recipe/build.sh

```diff
@@ -44,6 +44,7 @@ ${PYTHON} --version
 echo ""
 echo "show setuptools version"
 echo ""
+conda install -c anaconda pip
```

Comment:
This should be added in `requirements/host`

Reply:
Thanks---this was just screwing around, I think I misunderstood how the build environment works.  I might have a solution now... not clear yet...